File: src/my_project_typescript/easy_problems_typescript/from101to150/occurrencesAfterBigram.ts
function findOcurrences(text: string, first: string, second: string): string[] {

    let textList: Array<string> = text.split(" ")
    let lenText: number = textList.length
    let answer: Array<string> = []

    for (let i = 0; i < lenText-2; i++){
        if (textList[i] === first && textList[i+1] === second){
            answer.push(textList[i+2])
        }
    }

    return answer 
};



console.log(findOcurrences("alice is a good girl she is a good student",  "a",  "good"))

console.log(findOcurrences("we will we will rock you",  "we", "will"))

console.log(findOcurrences("alice is a good girl she is a good",  "a",  "good"))

File: src/my_project/easy_problems/from201to250/occurrences_after_bigram.py
from typing import List 

class Solution:
    def findOcurrences(self, text: str, first: str, second: str) -> List[str]:
        
        text_list = text.split()
        len_text: int = len(text_list)
        answer  = list()
        
        for i in range(len_text-2):
            
            
            if text_list[i] == first and text_list[i+1] == second:
                answer.append(text_list[i+2])
        
        
        return answer  
    
    
solution = Solution()

print(solution.findOcurrences(text = "alice is a good girl she is a good student", first = "a", second = "good"))

print(solution.findOcurrences(text = "we will we will rock you", first = "we", second = "will"))

print(solution.findOcurrences(text = "alice is a good girl she is a good", first = "a", second = "good"))


    
    
        